app/src/main/java/com/provider/service/impl/ItemServiceImpl.java

```diff
@@ -0,0 +1,100 @@
+package com.provider.service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import org.springframework.stereotype.Service;
+
+import com.provider.model.ItemRequestModel;
+import com.provider.model.ItemReturnModel;
+import com.provider.model.ItemReturnModelResult;
+import com.provider.model.SubItemRequestModel;
+import com.provider.model.SubItemReturnModel;
+import com.provider.persistence.entity.Item;
+import com.provider.persistence.entity.SubItem;
+import com.provider.persistence.repository.ItemRepository;
+import com.provider.persistence.repository.ProviderRepository;
+import com.provider.service.ItemService;
+import com.provider.service.ItemValidator;
+
+import jakarta.persistence.EntityManager;
+import jakarta.transaction.Transactional;
+import lombok.RequiredArgsConstructor;
+
+@Service
+@RequiredArgsConstructor
+public class ItemServiceImpl implements ItemService {
+    
+    private final ItemValidator itemValidator;
+
+    private final ItemRepository itemRepository;
+
+    private final ProviderRepository providerRepository;
+
+    private final EntityManager entityManager;
+
+    private SubItem createSubItem(SubItemRequestModel subItemRequestModel, Item item) {
+        SubItem subItem = new SubItem();
+        subItem.setTitle(subItemRequestModel.getTitle());
+        subItem.setDescription(subItemRequestModel.getDescription());
+        subItem.setPriceCents(subItemRequestModel.getPriceCents());
+        subItem.setStatus("view-only");
```

Comment:
Better to extract this as an enum, and use it on entities. Just need to create enum and jpa converter